lxd/db/query/marshal.go

```diff
@@ -0,0 +1,35 @@
+package query
+
+import (
+	"fmt"
+)
+
+type DBMarshaler interface {
```

Comment:
As this is inside the db/query package I think we can just call this Marshaler, like the Marsheler in the json package.

Reply:
Updated.